superset-frontend/src/views/CRUD/data/dataset/DatasetList.test.jsx

```diff
@@ -181,6 +182,14 @@ describe('DatasetList', () => {
       wrapper.find('[data-test="bulk-select-copy"]').text(),
     ).toMatchInlineSnapshot(`"3 Selected (2 Physical, 1 Virtual)"`);
   });
+
+  it('renders a SubMenu', () => {
+    expect(wrapper.find(SubMenu)).toExist();
+  });
+
+  it('renders a SubMenu with no tabs', () => {
+    expect(wrapper.find(SubMenu).props().tabs).toBeUndefined();
```

Comment:
same thought process here.

Reply:
undefined is correct since nothings should be passed to it, not even null.